### ISSUE TITLE
FLUX.1-Fill-dev running problem

### ISSUE THREAD

**Issue by pique2233** [black-forest-labs/flux#468]
My OS is Ubuntu20.04,CUDA version = 12.4,torch = 2.4.0,python=3.11 diffusers = 0.34.0,transformers = 4.53.2. When I checked with nvidia-smi, I found that there was no inference on the gpu, but it was still the problem from yesterday. The pipeline components had been loaded to 100%

Describe the bug
I don't know why, but the loading has remained stationary at this step. I'm running it on a 4090-24G graphics card. But it seems that it has been stuck since loading the pipe
(worldgen) lqz27@rise:~/.../WorldGen/flux$ python flux.py
Loading checkpoint shards: 100%|█████████████████| 2/2 [00:00<00:00, 149.45it/s]
Loading checkpoint shards: 100%|██████████████████| 3/3 [00:00<00:00, 95.33it/s]
Loading pipeline components...: 57%|███████▍ | 4/7 [00:00<00:00, 36.35it/s]You set add_prefix_space. The tokenizer needs to be converted from the slow tokenizers
Loading pipeline components...: 100%|█████████████| 7/7 [00:00<00:00, 26.22it/s]

Reproduction
import torch
from diffusers import FluxFillPipeline
from diffusers.utils import load_image

image = load_image("https://huggingface.co/datasets/diffusers/diffusers-images-docs/resolve/main/cup.png")
mask = load_image("https://huggingface.co/datasets/diffusers/diffusers-images-docs/resolve/main/cup_mask.png")

pipe = FluxFillPipeline.from_pretrained("black-forest-labs/FLUX.1-Fill-dev", torch_dtype=torch.bfloat16).to("cuda")
image = pipe(
prompt="a white paper cup",
image=image,
mask_image=mask,
height=1632,
width=1232,
guidance_scale=30,
num_inference_steps=50,
max_sequence_length=512,
generator=torch.Generator("cpu").manual_seed(0)
).images[0]
image.save(f"flux-fill-dev.png")
